EventRouter/network/eventServer.go

```diff
@@ -78,9 +69,9 @@ func (server *Server) listenIncomingConnections() {
 		conn, err := server.listener.Accept()
 		if err != nil {
 			log.Error("connection from factomd failed: %v", err)
+		} else {
```

Comment:
no else here: this improves the line of sight

Reply:
Fixed , I have to add a continue after the error, otherwise it's going to fire up a go-routine with a nil connection.